src/content-helper/editor-sidebar/smart-linking/component.tsx

```diff
@@ -470,7 +470,11 @@ export const SmartLinkingPanel = ( {
 			await processSmartLinks( generatedLinks );
 			setIsReviewModalOpen( true );
 		} catch ( e: any ) { // eslint-disable-line @typescript-eslint/no-explicit-any
-			let snackBarMessage = __( 'There was a problem generating smart links.', 'wp-parsely' );
+			const contentHelperError = new ContentHelperError(
```

Comment:
Since the error can be any type of object theoretically, we add some special handing to make sure it translates to a `ContentHelperError`.